db/warehouse/migrate/20220601122623_add_deleted_dat_to_ce_clients.rb

```diff
@@ -0,0 +1,32 @@
+class AddDeletedDatToCeClients < ActiveRecord::Migration[6.1]
```

Comment:
I think this was supposed to be `DeletedAt`, but humorously, it could also have been supposed to be `DeletedDate` :-)

Reply:
Thankfully the class name has nothing to do with the contents, since as you can see, this doesn't just add deleted_dat(s).